errors/unique_violation.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+type UniqueViolationError struct {
```

Comment:
If you, my dear reviewer, have another idea on how to pass such an error between layers in order not to use "magic strings" then I will be very grateful for your suggestion :)